### DISCUSSION TITLE
Flow Deployment Stucked On Building

### DISCUSSION THREAD

**Discussion by falahrumaria** [postmanlabs/postman-flows#440] [Defect]
Hi postman team, I am building a first flow, but it is stuck on "building" status. On the other hand, I try to deploy another flow but it is on "waiting" status. I suspect because of the other stuck deployment. The process deploy is running so slow while it is just a simple flow. After some time I closed the first flow tab, and open it again, then I see "failed" status `Sync:ESOCKETTIMEDOUT`. Can improve this flow feature? Be more real-time so users don't have to close and open it again to see the updated status? And if it is stuck or fail, can give us more detailed feedback what the cause is. Because I generally love the idea of this feature.

**Comment by viko42** [comment]
Which version of postman are you using ? Make sure you have the latest 10.10.8
